<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messageWindow</class>
 <widget class="QMainWindow" name="messageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contacts</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="contactList">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add contact</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="msgBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="contactMsgEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unreadBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Unread message</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>messageWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
